lib/dispatch-rider/callbacks/storage.rb

```diff
@@ -1,11 +1,15 @@
 module DispatchRider
   module Callbacks
+    # Storage for callbacks.
     class Storage
 
       def initialize
-        @callbacks = {}
+        @callbacks = Hash.new { |storage, key| storage[key] = [] }
```

Comment:
Should stuff this somewhere and call it "HashSet"  I think we do this SUPER often.